engine/lib/upgrades/2015062400-1.11.2-discussions_plugin-92f374deba4dc416.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Elgg 1.11.2 upgrade 2015062400
+ * discussions_plugin
+ *
+ * Discussions feature was pulled from groups plugin into its
+ * own plugin, so we need to update references of subtype
+ * 'groupforumtopic' into 'discussion'.
```

Comment:
This is diminutive compared e.g. to migration of comments from annotations to entities.

And it's trivial to fix the plugins. I really don't see this as such a bad thing.